apps/pkcs8.c

```diff
@@ -85,6 +87,7 @@ int pkcs8_main(int argc, char **argv)
     int nocrypt = 0, ret = 1, iter = PKCS12_DEFAULT_ITER;
     int informat = FORMAT_UNDEF, outformat = FORMAT_PEM, topk8 = 0, pbe_nid = -1;
     int private = 0, traditional = 0;
+    int saltlen = 0;
```

Comment:
Suggestion: use 128 here and print the default in the help.  Similar to what was done in #20466:

https://github.com/openssl/openssl/blob/a901b31e99442f087051ae7efdcbc9ad6e6a5b33/apps/enc.c#L95-L97

Reply:
Done